dashboard/config/programming_expressions/applab/button.json

```diff
@@ -10,5 +10,6 @@
       "name": "text"
     }
   ],
-  "syntax": "button(id, text)"
+  "syntax": "button(id, text)",
+  "video_key": "alg_6_variables"
```

Comment:
is this change to an applab block expected?

Reply:
oh oops 😅